minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -128,6 +130,21 @@ public void initConfiguration() throws IOException {
             mHostname, mWorkDirectory).entrySet()) {
       ServerConfiguration.set(entry.getKey(), entry.getValue());
     }
+    if (mJournalType == JournalType.UFS) {
+      ServerConfiguration.set(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.UFS);
+    } else {
+      ServerConfiguration.set(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.EMBEDDED);
+      ServerConfiguration.set(PropertyKey.MASTER_EMBEDDED_JOURNAL_ADDRESSES,
+          String.format("%s:%s", mHostname,
+              ServerConfiguration.get(PropertyKey.MASTER_EMBEDDED_JOURNAL_PORT)));
+      ServerConfiguration.set(PropertyKey.MASTER_RPC_ADDRESSES, String.format("%s:%s", mHostname,
```

Comment:
this shouldn't be needed since we've already set the embedded journal addresses

Reply:
Emmm, looks like both `EMBEDDED_JOURNAL_ADDRESSES` and `RPC_ADDRESSES` are not needed, since we set the `MASTER_HOSTNAME` and all the ports.